source/404.md

```diff
@@ -6,16 +6,14 @@ title: 404 Page Not Found
 
 # 404 Page Not Found
 
-The page was moved or never existed.  
-Use menu above to navigate or go to [home page](/).  
-If you noticed any site problem, typos, etc on any page, 
-please correct using "Edit on GitHub" button or check on <https://github.com/apache/struts-site>.
+The page was moved or never existed. You may use menu above to navigate or go to [home page](/).
+If you noticed any site problem (typos, etc on any page), you're able to correct it via [pull requesting](https://github.com/apache/struts-site/compare).
```

Comment:
The previous sentence was better (i.e. _`Edit on GitHub` button ..._) also navigating users to the compare is not a good idea, they first must create a branch (which can be done vie `Edit on Github`) to be able create a PR.